.prettierrc

```diff
@@ -0,0 +1,3 @@
+{
+	"useTabs": true
```

Comment:
I think this is generally good for accessibility / personal preference but as elm mandates the use of spaces, is this throwing a difference in where we maybe don't need one?

Reply:
I suppose I thought it would be nice if our JS was consistent across projects but it's not a big deal for me. There was talk of this becoming the default for prettier 3 but I didn't see it confirmed so that's probably not a good reason to stick to it.

Reply:
will revert if you feel strongly

Reply:
Do you?